doc/source/release/1.12.0-notes.rst

```diff
@@ -273,6 +273,14 @@ Deprecated features
 - All wavelet functions have been deprecated, as PyWavelets provides suitable
   implementations; affected functions are: ``signal.{daub, qmf, cascade,
   morlet, morlet2, ricker, cwt}``
+- ``scipy.integrate.trapz``, ``scipy.integrate.cumtrapz``, and ``scipy.integrate.simps`` have
+  been deprecated in favour of `scipy.integrate.trapezoid`, `scipy.integrate.cumulative_trapezoid`,
+  and `scipy.integrate.simpson` respectively and will be removed in SciPy 1.14.
+- The deprecation of the ``tol`` argument of ``scipy.sparse.linalg.{gcrotmk,lgmres,minres,tfqmr}``
+  is now consistent with the other iterative solvers and will be removed in SciPy 1.14
```

Comment:
I believe this is now correct @h-vetinari 
```suggestion
- The ``tol`` argument of ``scipy.sparse.linalg.{gcrotmk,lgmres,minres,tfqmr,bcg,bicstab,cg,cgs,gmres,qmr}``
  is now deprecated and will be removed in SciPy 1.14
```

Reply:
Let's pull forward the mention of `rtol` and alphabetize the solvers
```suggestion
- The ``tol`` argument of ``scipy.sparse.linalg.{bcg,bicstab,cg,cgs,gcrotmk,gmres,lgmres,minres,qmr,tfqmr}``
  is now deprecated in favour of ``rtol`` and will be removed in SciPy 1.14.
```